client/src/assets/icons/pencil_circle_green.svg

```diff
@@ -0,0 +1 @@
09)"/><rect x="81.198" y="90.204" width="26.492" height="8.704" transform="matrix(0.707,-0.707,0.707,0.707,-39.199,94.477)" fill="rgb(209,209,209)"/></g><g><g><path d="M 24.944 67 L 63.83 67 C 64.476 67 65 67.524 65 68.17 L 65 103.411 C 65 104.057 64.476 104.581 63.83 104.581 L 24.944 104.581 C 24.298 104.581 23.774 104.057 23.774 103.411 L 23.774 68.17 C 23.774 67.524 24.298 67 24.944 67 Z" style="stroke:none;fill:#49BD9E;stroke-linecap:square;stroke-miterlimit:2;"/><path d="M 24.944 67 L 63.83 67 C 64.476 67 65 67.524 65 68.17 L 65 103.411 C 65 104.057 64.476 104.581 63.83 104.581 L 24.944 104.581 C 24.298 104.581 23.774 104.057 23.774 103.411 L 23.774 68.17 C 23.774 67.524 24.298 67 24.944 67 Z" style="fill:none;stroke:#49BD9E;stroke-width:2;stroke-linecap:square;stroke-miterlimit:2;"/></g><text transform="matrix(0.668,0,0,0.668,31.273,110.963)" style="font-family:'Spinnaker';font-weight:400;font-size:60px;font-style:normal;fill:#ffffff;stroke:none;">+</text></g><g><rect x="87.892" y="-18.905" width="22.026" height="6" transform="matrix(0.707,-0.707,0.707,0.707,40.215,65.278)" fill="rgb(235,235,235)"/><polygon points="92.219,-4.799,86.125,-3.111,87.71,-9.233" fill="rgb(235,235,235)"/><path d="M 105.628 -26.87 L 107.172 -28.415 C 107.952 -29.195 109.22 -29.195 110 -28.415 L 111.415 -27 C 112.195 -26.22 112.195 -24.952 111.415 -24.172 L 109.87 -22.628 L 105.628 -26.87 Z" style="stroke:none;fill:#EBEBEB;stroke-linecap:square;stroke-miterlimit:2;"/></g><g><rect x="51.956" y="-48.42" width="19.857" height="5.409" transform="matrix(0.707,-0.707,0.707,0.707,50.452,30.369)" fill="rgb(255,255,255)"/><polygon points="55.857,-35.703,50.363,-34.182,51.792,-39.7" fill="rgb(255,255,255)"/><path d="M 67.945 -55.601 L 69.198 -56.854 C 69.978 -57.635 71.246 -57.635 72.026 -56.854 L 73.023 -55.858 C 73.803 -55.077 73.803 -53.81 73.023 -53.029 L 71.77 -51.776 L 67.945 -55.601 Z" style="stroke:none;fill:#FFFFFF;stroke-linecap:square;stroke-miterlimit:2;"/><ellipse vector-effect="non-scaling-stroke" cx="61.884615384615344" cy="-45.71590909090915" rx="22.88461538461536" ry="20.284090909090892" fill="none" stroke-width="2" stroke="rgb(255,255,255)" stroke-linejoin="miter" stroke-linecap="square" stroke-miterlimit="3"/></g><g><rect x="15.367" y="22.376" width="19.266" height="5.248" transform="matrix(0.707,-0.707,0.707,0.707,-10.355,25)" fill="rgb(73,189,158)"/><polygon points="19.151,34.715,13.821,36.191,15.208,30.836" fill="rgb(73,189,158)"/><path d="M 30.88 15.409 L 32.054 14.235 C 32.834 13.455 34.101 13.455 34.882 14.235 L 35.765 15.118 C 36.545 15.899 36.545 17.166 35.765 17.946 L 34.591 19.12 L 30.88 15.409 Z" style="stroke:none;fill:#49BD9E;stroke-linecap:square;stroke-miterlimit:2;"/><ellipse vector-effect="non-scaling-stroke" cx="25.000000000000004" cy="25.000000000000053" rx="22.203586793864318" ry="19.680451930925187" fill="none" stroke-width="2" stroke="rgb(73,189,158)" stroke-linejoin="miter" stroke-linecap="square" stroke-miterlimit="3"/></g></g></svg>
```

Comment:
It looks like the 'circle' is not perfectly round, or was your intention to be an ellipsis instead?
![Screenshot_20190702_094251](https://user-images.githubusercontent.com/4089035/60522138-0d20b280-9cae-11e9-92d0-09bb989bc38b.png)